rudder-rest/src/main/scala/com/normation/rudder/rest/settings/SettingsApi.scala

```diff
@@ -521,12 +495,88 @@ extends SettingsApi with Loggable {
     case Post(key :: Nil, req) => {
       import net.liftweb.json.JsonDSL._
       val data : Box[JValue] = for {
-        setting <- RestSetting(key)
+        setting <- settingFromKey(key)
         value   <- setting.setFromRequest(req)
       } yield {
         (key -> value)
       }
       RestUtils.response(restExtractor, "settings", Some(key))(data, req, s"Could not modify parameter '${key}'")("modifySetting")
     }
   }
+
+}
+
+class SettingsAPI8(
+    override val restExtractor        : RestExtractorService
+  , override val configService        : ReadConfigService with UpdateConfigService
+  , override val asyncDeploymentAgent : AsyncDeploymentAgent
+  , override val uuidGen              : StringUuidGenerator
+) ( override implicit val userService : UserService )
+extends SettingsApi with Loggable {
+
+
+  val allSettings =
```

Comment:
I would have put the list in the SettingsAPI in a val (baseSettings) and remove or add skipidentify

I would go for baseSettings to be api v9 list 

in v9 allSettings = baseSettings
in v8 allSettings = baseSettings :+ RestUseReverseDNS

Reply:
Make sense ! Thank you